src/App.tsx

```diff
@@ -1,223 +1,110 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { addTodo, getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { filterTodosByCompleted } from './helpers/helpers';
+import { FilterStatus } from './types/Filterstatus';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterStatus, setFilterStatus] = useState(FilterStatus.All);
+  const [newTitle, setNewTitle] = useState('');
+
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const completedTodos = todos.filter(todo => todo.completed === true);
+
+  const showError = useCallback((message: string) => {
+    setErrorMessage(message);
+
+    setTimeout(() => setErrorMessage(''), 3000);
+  }, []);
+
+  const uncompletedTodosAmount = useMemo(() => {
+    const uncompleted = todos.filter(todo => !todo.completed);
+
+    return uncompleted.length;
+  }, [todos]);
+
   useEffect(() => {
-    // focus the element with `ref={newTodoField}`
     if (newTodoField.current) {
       newTodoField.current.focus();
     }
-  }, []);
+
+    if (user) {
+      getTodos(user.id)
+        .then(setTodos)
+        .catch(() => showError('Unable to load todos'));
```

Comment:
It is better to create ENUM for error messages
```suggestion
        .catch(() => showError(TodoErrors.UnableToLoad));
```